src/main/java/seedu/address/model/accommodation/Accommodation.java

```diff
@@ -0,0 +1,111 @@
+package seedu.address.model.accommodation;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Date;
+import java.util.Objects;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Name;
+
+/**
+ * Represents an Accommodation in the travel plan.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Accommodation {
+
+    private final Name name;
+    private final Date startDate;
+    private final Date endDate;
+    private final Cost cost;
+    private final Address address;
```

Comment:
I think can change address to location class? Since our user guide its location class. Can just import from activity.